frontend/src/views/web/challenge/edit-challenge/edit-challenge-date.html

```diff
@@ -0,0 +1,42 @@
+<section class="ev-md-container text-center rm-overflow-y">
+    <div class="row">
+        <div class="col s12 m12">
+            <div class="ev-card-body pd-20 font-size-16">
+                <form name="editChallengeDateForm" ng-submit="challenge.editChallengeDate(editChallengeDateForm.$valid)">
+                    <div class="pass-date">Edit Challenge Date</div>
```

Comment:
Please add it appropriately for both start and end dates like `Edit challenge start date` and `Edit challenge end date`